### DISCUSSION TITLE
Vanjaro on multi portal instance

### DISCUSSION THREAD

**Discussion by kingjordan** [vanjarosoftware/Vanjaro.Platform#2298] [General]
Hello,

I have a dnn instance running in azure that hosts a number of very simple sites as different portals in the same DNN instance.
If I install vanjaro as an extension, is it possible to migrate websites one at a time or only enable it for certain portals?

**Comment by meetmandeep** [comment]
Absolutely! You can also migrate individual pages within a single portal. This allows you the flexibility to migrate as needed and the ability to start new pages with Vanjaro or native DNN functionality.

**Comment by kingjordan** [comment]
great news, thanks
